src/fvm_multicell.hpp

```diff
@@ -233,8 +247,8 @@ class fvm_multicell {
 //////////////////////////////// Implementation ////////////////////////////////
 ////////////////////////////////////////////////////////////////////////////////
 
-template <typename T, typename I>
-void fvm_multicell<T, I>::compute_cv_area_unnormalized_capacitance(
+template <typename T, typename I, typename DivPolicy>
+void fvm_multicell<T, I, DivPolicy>::compute_cv_area_unnormalized_capacitance(
     std::pair<size_type, size_type> comp_ival,
     const segment* seg,
     index_type &parent)
```

Comment:
I don't think you need the `using util::left/right` statements below (I can't seem to leave a comment on the lines themselves)